File: examples/javascript/tasks/fillMask/fillMask.mjs
import Bytez from "bytez.js";

const client = new Bytez("YOUR_BYTEZ_KEY_HERE");

const model = client.model("almanach/camembert-base");

await model.load();

const { output: sequenceObjects } = await model.run(
  "The capital of France is <mask>."
);

for (const sequenceObject of sequenceObjects) {
  // depending on the model, there may be additional props returned
  console.log(sequenceObject);

  const { sequence, score, token, token_str } = sequenceObject;

  console.log({ sequence, score, token, token_str });
}

File: examples/python/tasks/fill_mask/fill_mask.py
from bytez import Bytez

client = Bytez("YOUR_BYTEZ_KEY_HERE")

model = client.model("almanach/camembert-base")

model.load()

input_text = "The capital of France is <mask>."

result = model.run(input_text)

sequence_objects = result.get("output")


for sequence_object in sequence_objects:
    # Depending on the model, there may be additional props returned
    print(sequence_object)

    sequence = sequence_object["sequence"]
    score = sequence_object["score"]
    token = sequence_object["token"]
    token_str = sequence_object["token_str"]

    print(
        {"sequence": sequence, "score": score, "token": token, "token_str": token_str}
    )
